warehouse/manage/views.py

```diff
@@ -279,19 +279,41 @@ def _key(project):
             return project.releases[0].created
         return project.created
 
-    projects_owned = set(
-        project.name
-        for project in (
-            request.db.query(Project.name)
-            .join(Role.project)
-            .filter(Role.role_name == "Owner", Role.user == request.user)
-            .all()
+    projects_owned = (
+        request.db.query(Project)
+        .join(Role.project)
+        .filter(Role.role_name == "Owner", Role.user == request.user)
+        .subquery()
+    )
```

Comment:
I'd prefer if the `projects_owned` variable wasn't being overwritten later on in this function, perhaps this could be called "user_projects_query" or something like that?

Reply:
Changed `projects_owned ` to `user_projects_query `